tutorials/notebooks/mct_features_notebooks/keras/example_keras_xquant.ipynb

```diff
@@ -0,0 +1,208 @@
+{
+  "nbformat": 4,
+  "nbformat_minor": 0,
+  "metadata": {
+    "colab": {
+      "provenance": []
+    },
+    "kernelspec": {
+      "name": "python3",
+      "display_name": "Python 3"
+    },
+    "language_info": {
+      "name": "python"
+    }
+  },
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "source": [
+        "# Quantization Troubleshooting with XQuant\n",
+        "\n",
+        "[Run this tutorial in Google Colab](https://colab.research.google.com/github/sony/model_optimization/blob/main/tutorials/notebooks/mct_features_notebooks/keras/example_keras_xquant.ipynb)\n",
+        "\n",
+        "This notebook demonstrates the process of generating an xquant report. The report provides valuable insights such as histograms and the similarity metrics between the original float model and the quantized model, which can be visualized using TensorBoard.\n",
```

Comment:
"The report provides valuable insights"
maybe add
"regarding the quality and success of the quantization".
and then in a the next sentence:
"This insights include histograms, similarity... in key points of the model..."
"The report can be visualized..."